### ISSUE TITLE
Go to any location from fantasy gets lost

### ISSUE THREAD

**Issue by sonic306** [Inix3K/TinTin#10]
When i am located in POL or any other "room" in that zone issuing a "go cot" will end up in a Cottage next to the necromancers guild portal

**Comment by lamnguyen2020** [comment]
yeah, when you're inside an area, go will match with closest thing inside the area.  so, go pol, or go magic.  Same thing for inside chaos, if you go chaos, it'll take you to the bar, go coc works.

**Comment by Inix3K** [comment]
go is alias for '#map run {roomnote} {%0%*}'  The %* at the end is a wildcard to allow for using partial names.  So yes, it'll match whatever starts with what you send that is the closest.  Haven't run into the 'cottage' issue, but it makes sense.  You can use 'go shop' to get to cancer, one east of cot, or '#map run 1' since cot is roomnumber 1.  The other solution would be to enforce full names, but I feel the usefulness of the wildcard outweighs the occasional closer matches.
